### DISCUSSION TITLE
Custom command failed

### DISCUSSION THREAD

**Discussion by bastien70** [deployphp/deployer#2312] [Help needed]
Deployer version : v6.8
Hello, in my Symfony 5.1 project I am trying to declare my own task which I would like to use when deploying. I created it like this:

```php
task('update_database', function () {
    if($_ENV['DATABASE_URL'])
    {
        cd('{{release_path}}');
        writeln('<info>Updating database</info>');
        run('{{bin/php}} {{bin/console}} d:d:c --if-not-exists');
        run('{{bin/php}} {{bin/console}} doctrine:schema:update --force');
    }
})->desc('Make database updates');
```

The "cd" and the "writeln" work. But every time I try to run a command it always fails

![custom task](https://user-images.githubusercontent.com/53140475/100590085-38209300-32f4-11eb-8329-2ba2858d6711.PNG)

I tried with just the command `php bin/console` instead of using `{{bin/php}} {{bin/console}}`.

I even tried a silly `php bin/console cache: clear` command to see, but same error.

Yet I am in the right directory.. I followed this tutorial https://www.codepicky.com/php-automatic-deploy/ 

And I do the exact same thing though

**Comment by antonmedv** [comment]
Difficult to say. You neet to investigate and try to understand what is going wrong. Now everything looks alright.

**Reply by bastien70** [reply]
I connected to my server, I went to the last release of my project, and when I tried commands, absolutely nothing was happening. No mistake, but they didn't do anything. Even if I try the command to create an entity, it is "ignored". The -vvv flag didn't give anything either

Here is what I have in the console:

![erreurserveur](https://user-images.githubusercontent.com/53140475/100596203-3064ec80-32fc-11eb-8e18-a3d67c94cbc0.PNG)

But if i'm going to the first release, commands are OKAY


EDIT : Okay I found the error

**Reply by antonmedv** [reply]
What error?

**Reply by bastien70** [reply]
There was a bug the last time I wanted to deploy. Suddenly, the last release was not complete, the vendors were therefore not installed. So when I subsequently tried the order, it was obviously not going to work because the vendors allowing this were missing
